### PR TITLE
Create project setup issue template.

### PR DESCRIPTION
**Pull request by adamzimmermann**
## Description

Adds a project setup issue template and amends the launch issue template.

## Motivation / Context
We want to standardize the project setup process to ensure no steps are forgotten.

## Testing Instructions / How This Has Been Tested
N/A

## Screenshots
N/A

## Documentation
This is the documentation.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/launch.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ please update this ticket description to include references to those tickets.
 - [ ] Determine where the starting database will come from.
 - [ ] Determine who manages DNS.
 - [ ] Determine the caching strategy.
+- [ ] Take screenshots of "before" site.
+- [ ] Add site to [web.archive.org](http://web.archive.org/).
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/launch.md:24]: This sounds good, but what was the catalyst for this? Was there a time we did not do this and we ended up wishing we had?

File: .github/ISSUE_TEMPLATE/launch.md [hunk 2]
```diff
@@ -95,6 +97,10 @@ please update this ticket description to include references to those tickets.
       if a CDN or reverse-proxy is being used.
 - [ ] Run performance tests with a tool such as Calibre, Lighthouse, or
       PageSpeed Insights and fix the issues and/or log post launch tickets.
+- [ ] Configure field help/descriptive text.
+- [ ] Optimize media creation/referencing experience.
+- [ ] Verify that Metatags & JSON-LD are configured.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/launch.md]:
```suggestion
- [ ] Verify that meta tags & JSON-LD are configured.
```

https://www.w3schools.com/tags/tag_meta.asp

File: .github/ISSUE_TEMPLATE/launch.md [hunk 3]
```diff
@@ -95,6 +97,10 @@ please update this ticket description to include references to those tickets.
...
+- [ ] Verify that images are being served in a performant manner.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/launch.md]:
```suggestion
- [ ] Verify that images are being served in a efficient manner.
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+name: Setup
+about: Prepare for development with some basic project setup.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]:
```suggestion
about: Prepare for development with basic project setup.
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+title: ''
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]:
```suggestion
title: '[ prep ] '
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+labels: ''
+assignees: ''
+
+---
+
+Please check off line-items as they are completed and leave notes if necessary.
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`). If child tickets are created for any line-item,
+please update this ticket description to include references to those tickets.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]:
```suggestion
Please check off line-items as they are completed and leave notes if necessary.
If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
(e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for any
line-item, please update this ticket description to include references to those
tickets.
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+- [ ] Create a README.
+- [ ] Set up Tugboat.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]:
```suggestion
- [ ] Set up [Tugboat](https://dashboard.tugboat.qa/).
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Set up Dependabot.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]:
```suggestion
- [ ] Set up Dependabot ([example](https://github.com/ChromaticHQ/chromatichq.com/blob/master/.github/dependabot.yml)).
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Set up PHP code style checks.
+- [ ] Set up front-end linting.
+- [ ] Set up GitHub Actions to run code style checks on YML, shell scripts, and
+      execute PHP checks/front-end linting tools.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]:
```suggestion
- [ ] Set up GitHub Actions (example)[https://github.com/ChromaticHQ/chromatichq.com/tree/master/.github/workflows]
      to run code style checks on YML, shell scripts, and execute PHP
      checks/front-end linting tools.
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Set up `settings.php` file infrastructure.
+- [ ] Set up Composer file.
+- [ ] Set up front-end tooling.
+- [ ] Set up Lando file.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]:
```suggestion
- [ ] Set up Lando file ([example](https://github.com/ChromaticHQ/chromatichq.com/blob/master/.lando.yml)).
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Set up a production environment.
+- [ ] Set up nightly database backups.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]:
```suggestion
- [ ] Set up nightly database backups to S3.
```

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Set up development servers.
```
Comment (markdorison) [inline .github/ISSUE_TEMPLATE/setup.yml]: What do we mean by this?

**Review by markdorison — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/launch.md (hunk 1 above)
Comment (adamzimmermann) [reply .github/ISSUE_TEMPLATE/launch.md:24]: Good question. I just found it in the list and carried it over. I presume it was to aid us in creating case studies?

File: .github/ISSUE_TEMPLATE/setup.yml (hunk 6 above)
Comment (adamzimmermann) [reply .github/ISSUE_TEMPLATE/setup.yml]: I stole this text from the `launch.yml` template. Want me to make this change there too?

File: .github/ISSUE_TEMPLATE/setup.yml (hunk 8 above)
Comment (adamzimmermann) [reply .github/ISSUE_TEMPLATE/setup.yml]: Good call.

File: .github/ISSUE_TEMPLATE/setup.yml (hunk 12 above)
Comment (adamzimmermann) [reply .github/ISSUE_TEMPLATE/setup.yml]: We can probably remove this since we don't really use development servers now.

Comment (adamzimmermann) [reply .github/ISSUE_TEMPLATE/setup.yml]: Removed.

File: .github/ISSUE_TEMPLATE/setup.yml (hunk 6 above)
Comment (markdorison) [reply .github/ISSUE_TEMPLATE/setup.yml]: Sure!

**Review by markdorison — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/setup.yml (hunk 6 above)
Comment (adamzimmermann) [reply .github/ISSUE_TEMPLATE/setup.yml]: Done.

**Comment by adamzimmermann** [comment]
@agarzola I'll wait for your approval here too, as I want to make sure the front-end bits are comprehensive.

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Set up [Tugboat](https://dashboard.tugboat.qa/).
+- [ ] Set up Dependabot ([example](https://github.com/ChromaticHQ/chromatichq.com/blob/master/.github/dependabot.yml)).
+- [ ] Set up PHP code style checks.
+- [ ] Set up front-end linting.
+- [ ] Set up GitHub Actions (example)[https://github.com/ChromaticHQ/chromatichq.com/tree/master/.github/workflows]
+      to run code style checks on YML, shell scripts, and execute PHP
+      checks/front-end linting tools.
+- [ ] Set up `settings.php` file infrastructure.
+- [ ] Set up a `composer.json` file.
+- [ ] Set up front-end tooling.
```
Comment (agarzola) [inline .github/ISSUE_TEMPLATE/setup.yml:26]: Before front-end tooling is even added, we need to _have_ a theme. Likewise, it is important to build theme assets in remote environments, so we should add a line items for that as well.
```suggestion
- [ ] Create a theme at `web/themes/custom/THEME_NAME`.
- [ ] Set up front-end tooling. This should provide:
    - [ ] A task to build the theme’s static assets.
    - [ ] Development workflow tasks to build assets automatically, watch for changes, refresh the browser automatically, etc.
    - [ ] CSS and JS linting tasks, as well as any other testing tasks that relevant to the project.
- [ ] Document system-level requirements and step(s) to build theme assets in remote environments in the theme’s README.
- [ ] Update build scripts (used in Tugboat, production, and elsewhere) to install front-end dependencies and build theme assets.
- [ ] Update GitHub Actions to run any front-end linting and/or testing tasks.
```

Thoughts?

File: .github/ISSUE_TEMPLATE/setup.yml [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for
+any line-item, please update this ticket description to include references to
+those tickets.
+
+- [ ] Create a README.
+- [ ] Set up [Tugboat](https://dashboard.tugboat.qa/).
+- [ ] Set up Dependabot ([example](https://github.com/ChromaticHQ/chromatichq.com/blob/master/.github/dependabot.yml)).
+- [ ] Set up PHP code style checks.
+- [ ] Set up front-end linting.
```
Comment (agarzola) [inline .github/ISSUE_TEMPLATE/setup.yml]: This is part of the “Set up front-end tooling” step.

Comment (adamzimmermann) [reply .github/ISSUE_TEMPLATE/setup.yml]: Removing. Thanks for calling this out and adding the fantastic suggestion!

**Comment by adamzimmermann** [comment]
Ready for final review.

**Review by markdorison — APPROVED** [review]

**Comment by adamzimmermann** [comment]
@agarzola I'm happy to adjust this more, but I added your suggestions, so I'm going to go ahead and merge for now.

**Pull request merged**